README.md

```diff
@@ -16,16 +16,63 @@ bower install markdown-it-link-attributes --save
 ```js
 var md = require('markdown-it')()
 var mila = require('markdown-it-link-attributes')
+
+// Config
+const options = {
+  pattern: RegExp, // Default: undefined
+  ignoreRelative: Boolean, // Default: false
+  ...attrs // Attributes to assign to the link
+}
+
+md.use(mila, options)
+// or
+md.use(mila, [ ...options ])
```

Comment:
It would probably be good to document what configs take precedence when there are multiple potential matches.